core/tests/protractor_utils/editor.js

```diff
@@ -112,6 +112,11 @@ var expectCurrentStateToBe = function(name) {
   expect(_getStateName()).toMatch(name);
 };
 
+var isOnFeedbackTab = function() {
```

Comment:
Ah -- I originally thought we would be returning element(...).isPresent(), but I suppose maybe that doesn't play well with Protractor.

Can we rename to expectCurrentTabToBeFeedbackTab(), and have this not return anything?

Also, technically, should you also include this in the exports list at the bottom of the file?

Reply:
Yup forgot that, thanks. Done.